packages/ui-components/src/components/global/ToastRenderer.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div
     aria-live="assertive"
-    class="pointer-events-none fixed top-0 right-0 left-0 bottom-0 flex items-end px-4 py-6 mt-10 sm:items-start sm:p-6 z-50"
+    class="pointer-events-none fixed top-0 right-0 left-0 bottom-0 flex items-end px-4 py-6 mt-10 sm:items-start sm:p-6 z-[60]"
```

Comment:
No huge fan of doing it this way, for it's the only way to have the toasts above the dialogs for now

Reply:
I don't have an issue with this. If there are meaningless z-index's it can get annoying, but we actually need all these levels of elevation, so this is fine for me.